back-end/photo_app/models.py

```diff
@@ -1,21 +1,23 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.forms import CharField
```

Comment:
We should *not* be importing `CharField` from `django.forms`, it should be coming from `django.db.models`.